conf/logback.xml

```diff
@@ -22,6 +22,7 @@
     </appender>
 
     <logger name="play" level="INFO" />
+    <logger name="com.google.api.client.http.HttpTransport" level="WARN" />
```

Comment:
Would it make it clearer to set it to `level="OFF"`

Reply:
https://github.com/search?l=XML&q=org%3Aguardian+com.google.api.client.http.HttpTransport&type=Code has a mix of `OFF` and `WARN`. I suppose it is possible that there is some logging that is useful.  The noise is the `INFO` level messages. I don't have strong feelings on this.

Reply:
Makes sense